macros/JsSidebar.ejs

```diff
@@ -34,8 +34,8 @@ var text = mdn.localStringMap({
     'Basics': 'JavaScript basics',
     'First_steps': 'JavaScript first steps',
     'Building_blocks': 'JavaScript building blocks',
-    'Intermediate': 'Intermediate',
     'Introducing_objects': 'Introducing JavaScript objects',
+    'Intermediate': 'Intermediate',
```

Comment:
I'm not sure why this switches the order for en-US but not for the other locales.